test/apps/iodemo/io_demo.cc

```diff
@@ -1047,10 +1053,21 @@ class P2pDemoCommon : public UcxContext {
     BufferMemoryPool<Buffer>         _data_chunks_pool;
     static status_t                  _status;
     const uint32_t                   _iov_buf_filler;
+public:
+    static BufferMemoryPool<Buffer>  _bogus_chunks_pool;
```

Comment:
can we initialize _bogus_buf it with default values instead of using _bogus_chunks_pool?

Reply:
i removed _bogus_chunks_pool but not sure the way i did is what you wanted.

Reply:
@yosefe can you elaborate what "default values" mean